.github/workflows/publish.yml

```diff
@@ -88,13 +88,20 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Setup CI
-        uses: input-output-hk/catalyst-ci/actions/setup@master
+        uses: input-output-hk/catalyst-ci/actions/setup@fix/ci-cli-build
         with:
-          cli_version: ${{ inputs.ci_cli_version }}
           configure_registries: "false"
-          earthly_skip_install: "true"
+          aws_ecr_registry: ${{ inputs.aws_ecr_registry }}
+          aws_role_arn: ${{ inputs.aws_role_arn }}
```

Comment:
didn't get why we need path these arguments as ` aws_ecr_registry`, ` aws_role_arn`, `aws_region` etc.   for the `setup` action, can you elaborate on this ?

Reply:
If I'm not mistaken, it is required for running Earthly remote